src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php

```diff
@@ -41,7 +41,7 @@ class ExceptionCaster
         \E_USER_ERROR => 'E_USER_ERROR',
         \E_USER_WARNING => 'E_USER_WARNING',
         \E_USER_NOTICE => 'E_USER_NOTICE',
-        \E_STRICT => 'E_STRICT',
+        2048 => 'E_STRICT',
```

Comment:
should be dependent on the presence of the constant IMO (that way we can more easily identify parts of the code to be cleaned up once support for older PHP versions is going to be dropped)

Reply:
I think there are few chances to forget it with the plain string next to it. Also, it is stored in a static property in a static class. It also requires extra logic wherever the static property is called, raising chances of introducing a bug if a new usage occurrence of the static property is introduced :confused: